### PR TITLE
makefile: remove bazel_check before ut

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close #xxx

Problem Summary:

avoid probably failure in the bazel_check 

### Check List

Tests 

- [x] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by xhebox — APPROVED** [review]

**Review by zimulala — COMMENTED** [review]
Why would you do that? Could you describe it briefly in PR?

**Comment by hawkingrei** [comment]
> Why would you do that? Could you describe it briefly in PR?

updated.

File: Makefile [hunk 1]
```diff
@@ -493,7 +493,7 @@ bazel_test: failpoint-enable bazel_prepare
 		-//tests/globalkilltest/... -//tests/readonlytest/... -//br/pkg/task:task_test -//tests/realtikvtest/...
 
 
-bazel_coverage_test: check-bazel-prepare failpoint-enable bazel_ci_simple_prepare
+bazel_coverage_test: failpoint-enable bazel_ci_simple_prepare
```
Comment (zimulala) [inline Makefile:496]: `check dev` also has this check, so we can remove the check here

**Review by zimulala — APPROVED** [review]
LGTM

**Pull request merged**